pkg/testing/integration_test.go

```diff
@@ -259,8 +259,8 @@ func (s *IntegrationSuite) TestRun(c *C) {
 		err = a.Reset(ctx)
 		c.Assert(err, IsNil)
 
-		count, err := a.Count(ctx)
-		c.Assert(err, IsNil)
+		count, _ := a.Count(ctx)
+		// c.Assert(err, IsNil)
```

Comment:
missed while cleanup?

Reply:
After reset call, the schema won't be present in cases where we are not using default schema. There, the Count would always return 0, err. So removed this Assert.

Reply:
why does it return an error? We should not get an error if count is 0.

Reply:
Earlier we used to initialize the schema in Reset call so count would always be successful. Now, in cases where we there is no table, the count query would fail but we still return 0 there.

Reply:
Ideally, Count() should not return an error because having no data present doesn't mean the count query has failed. It could be misleading. We need to refactor count() implementation accordingly. For now, we can just remove the validation from here.

Reply:
Right, removed the validation for now.